targets/mbedos5/jerryscript-mbed/jerryscript-mbed-library-registry/source/wrap_tools.cpp

```diff
@@ -0,0 +1,79 @@
+/* Copyright 2014-2015 Samsung Electronics Co., Ltd.
+ * Copyright (c) 2016 ARM Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "jerryscript-mbed-library-registry/wrap_tools.h"
+
+bool jsmbed_wrap_register_global_function(const char* name, jerry_external_handler_t handler) {
+    jerry_value_t global_object_val = jerry_get_global_object();
+    jerry_value_t reg_function = jerry_create_external_function(handler);
+
+    bool is_ok = true;
+
+    if (!(jerry_value_is_function(reg_function)
+        && jerry_value_is_constructor(reg_function))) {
+        is_ok = false;
+        LOG_PRINT_ALWAYS("Error: create_external_function failed !!!\r\n");
```

Comment:
Also a bit misleading error message missing the "jerry_" prefix.

(BTW: Any intention to get error messages even more consistent? Current strings use "ERROR" vs "Error" and  "!!!" vs "!" vs "." vs nothing and "\r\n" vs "\n\r" vs "\n".)